docs/user_guide/09_configuration_file.md

```diff
@@ -104,7 +106,7 @@ Most commonly, `cpu`, `gpu` or `tpu` as well as other [options](https://pytorch-
 
 If `gpu`, it can be helpful to specify the data parallelization strategy. This can be done using the `strategy` arg in `main.create_trainer()`
 
-```
+```python
 model.create_trainer(logger=comet_logger, strategy="ddp")
```

Comment:
Since this is a new section we should add the import here as well and switch from `model` to `m` for consistency